requirements/default.txt

```diff
@@ -2,3 +2,6 @@ typing_extensions; python_version<"3.10"
 amqp>=5.1.1,<6.0.0
 vine
 backports.zoneinfo[tzdata]>=0.2.1; python_version < '3.9'
+# due to this bug https://github.com/docker/docker-py/issues/3256
+# we need to hard-code the version of requests
+requests<2.32.0
```

Comment:
good catch!

Reply:
This bug is causing such a huge mess, sigh...

Reply:
We need to extract whatever handling we do with this requests/docker-py bug to a different PR.
I don’t want it to clutter the PR and specifically not this one.

Reply:
@awmackowiak @auvipy
https://github.com/celery/kombu/pull/2011